src/signals-mach.c

```diff
@@ -223,6 +224,10 @@ static void jl_throw_in_thread(int tid, mach_port_t thread, jl_value_t *exceptio
 
 static void segv_handler(int sig, siginfo_t *info, void *context)
 {
+    if(getenv("LD_LIBRARY_PATH")) {
```

Comment:
Can't use getenv in a signal handler. This check should happen some time at init and stash it's result in a static volatile.

Reply:
Alright, I'm now setting `ld_library_path_is_set` in `src/init.c`, but I'm not sure how to make sure that value is visible inside `src/signals-mach.c` and `src/signals-unix.c`.